sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Scenario/SqlDatabaseTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.Sql.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Sql.Tests.Scenario
+{
+    public class SqlDatabaseTests : SqlManagementClientBase
```

Comment:
The CI failed due to the test failure, please re-record the tests here. Two things to note are:

- Please record all tests at once. `SqlDatabaseApiTests` and `SqlDatabaseApiTestsWithEnclaves` share the common `GlobalSetUp`. Record the tests one by one will cause the record corresponding to GlobalSetUp to be rewritten and all previous tests will fail in playback mode.
- Please also record the tests both with [sync and async methods](https://github.com/Azure/azure-sdk-for-net/blame/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#L59), there is no sync method test recording.

Reply:
Thanks @Yao725 , I have re-recorded the tests and added both the sync and async methods.